examples/basic-chat.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Basic Chat — Networked-Aframe</title>
+    <meta name="description" content="Basic Example — Networked-Aframe" />
+
+    <script src="https://aframe.io/releases/1.4.2/aframe.min.js"></script>
+
+    <!--   NAF basic requirements   -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.5.0/socket.io.slim.js"></script>
+    <script src="/easyrtc/easyrtc.js"></script>
+    <script src="/dist/networked-aframe.js"></script>
+
+    <!--    used for flying in this demo  -->
+    <script src="https://cdn.jsdelivr.net/gh/c-frame/aframe-extras@7.0.0/dist/aframe-extras.controls.min.js"></script>
+
+    <!--   used for the pretty environment   -->
+    <script src="https://unpkg.com/aframe-environment-component@1.3.2/dist/aframe-environment-component.min.js"></script>
+
+    <!--   used to prevent players from spawning on top of each other so much  -->
+    <script src="/js/spawn-in-circle.component.js"></script>
+
+    <style>
+      .chat {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 300px;
+
+        overflow: hidden;
+
+        z-index: 100;
+      }
+
+      #input {
+        width: 70%;
+        padding: 10px;
+        border: none;
+        outline: none;
+        border-bottom: 1px solid #ccc;
+        box-sizing: border-box;
+      }
+
+      #send-btn {
+        width: 30%;
+        padding: 10px;
+        border: none;
+        background-color: #4caf50;
+        color: #fff;
+        cursor: pointer;
```

Comment:
You shouldn't need to specify the cursor type, that's the default for buttons.

Reply:
The default with buttons is 
`cursor: default;`
correct me if i am wrong

Reply:
Yes, and the default is pointer, right? like for links. If I comment it I don't see any change.

Reply:
i was seeing pointer but after commenting i saw default which is simple cursor

Reply:
My bad, sorry, you're right.